redi-connect-front/src/components/atoms/FormSelect.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { Form } from 'react-bulma-components'
+
+interface Props {
+  name: string
+  items: string[]
+  placeholder: string
+  label?: string
+  multiselect?: boolean
+  customOnChange?: Function
+}
+
+function FormInput (props: any) {
+  const {
+    name,
+    items,
+    placeholder,
+    label,
+    customOnChange,
+    values,
+    handleChange,
+    setFieldTouched,
+    multiselect,
+    isSubmitting,
+    touched,
+    errors,
+    disabled
+  } = props
+
+  const onChange = (name: any, e: any) => {
+    e.persist()
+    setFieldTouched(name, true, false)
+    handleChange(e)
+  }
+
+  const handleOnChange = customOnChange || onChange
+
+  // const hasError = !!touched[name] && !!errors[name]
+
+  return (
+    <Form.Field>
+      {label && <Form.Label size="small">
+        {label}
+      </Form.Label>}
+
+      <Form.Control>
+        <Form.Select
+          name={name}
+          className="is-fullwidth"
+          value={values[name]}
+          multiple={multiselect}
+          onChange={handleOnChange.bind(null, name)}
```

Comment:
Do we actually need to still use `bind` here? I doubt it, since `onChange` isn't using `this`

Reply:
not really. fixing

Reply:
great